openssh/USAGE.md

```diff
@@ -0,0 +1,128 @@
+## Purpose 
+
+This is an [opensshd](https://https.openssh.com) docker image based on the [OQS OpenSSH 7.9 fork](https://github.com/open-quantum-safe/openssh), which allows ssh to quantum-safely negotiate session keys and use quantum-safe authentication with algorithms from the [Post-Quantum Cryptography Project by NIST](https://csrc.nist.gov/projects/post-quantum-cryptography).
+
+This image has a built-in non-root user to permit execution without particular [docker privileges](https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities). This is necessary as logging in as root in ssh is not recommended practice. But it is worth to note that this user, per default called `oqs`, is not set as the default user when the image starts (which would be done with `USER oqs` in the [Dockerfile](Dockerfile)). The reason for that is that the the start up script needs root permissions to generate all host keys and start the sshd service. This means that when executing a command as the user `oqs`, the `docker exec` command needs to be used together with the option `--user oqs`.
+
+
+## Quick start 
+
+How to set up your quantum-safe OpenSSH is described in the corresponding [README.md](README.md).
+
+## Limitations
+
+For limitations of the post-quantum algorithms themselves, refer to [this](https://github.com/open-quantum-safe/openssh#limitations-and-security).
+
+## Slightly more advanced usage options
+
+### Change install location
+
+The OQS-OpenSSH binaries, configuration files and host keys are located in the default install location `/opt/oqs-ssh/`. This location can be changed at build time with `--build-arg INSTALL_DIR=</path/to/new/location>`.
+
+### Access the man pages of oqs-ssh
+
+Man pages for oqs-ssh are installed in `<INSTALL_DIR>/share/man` and can be viewed by for example `man -l <INSTALL_DIR>/share/man/man1/ssh.1` or `man -l <INSTALL_DIR>/share/man/man5/ssh.5`. Note that those man pages are **not** different from the original ssh man pages. So it might be easier to consult them on your local system or the internet. 
+
+Direct links to man pages: [ssh(1)](https://linux.die.net/man/1/ssh), [sshd(8)](https://linux.die.net/man/8/sshd), [ssh_config(5)](https://linux.die.net/man/5/ssh_config), [sshd_config(5)](https://linux.die.net/man/5/sshd_config)
+
+## Seriously more advanced usage options
```

Comment:
I would drop the value-judgement and just call this section "Advanced usage options" :)

Reply:
I blatantly stole that from @baentsch's [nginx USAGE.md](https://github.com/open-quantum-safe/oqs-demos/blob/main/nginx/USAGE.md#seriously-more-advanced-usage-options). I liked the humor and stuck with it 😃

Reply:
I didn't realize @baentsch was using it :) All good then, if we have precedent!